### PR TITLE
Fix Mito Report

### LINKED ISSUE #1352: Not possible to open mtDNA report
It is not possible to open mtDNA report in the cases analyzed after update of Scout.

### PR DESCRIPTION
**Pull request by northwestwitch**
Bug fix #1352

**How to test**:
1. Make sure that mitochondrial report for case  https://scout-stage.scilifelab.se/cust002/F0009130-validation doesn't work. It triggers an error
1. Install this branch on stage and try to download report for the same case

**Expected outcome**:
Report should be downloadable and look fine

**Review:**
- [x] code approved by @moonso 
- [x] tests executed by @moonso

### CODE REVIEW

**Comment by northwestwitch** [comment]
Whoever has time. It would be nice to include this in the release of today..

**Comment by moonso** [comment]
Have you installed on stage?

File: scout/export/variant.py [hunk 1]
```diff
@@ -135,13 +135,14 @@ def export_mt_variants(variants, sample_id):
         line.append(str(position)+change)
         genes = []
         prot_effect = []
-        for gene in variant.get('genes'):
-            genes.append(gene.get('hgnc_symbol',''))
-            for transcript in gene.get('transcripts'):
-                if transcript.get('is_canonical') and transcript.get('protein_sequence_name'):
-                    prot_effect.append(urllib.parse.unquote(transcript.get('protein_sequence_name')))
-        line.append(','.join(prot_effect))
+        if variant.get('genes'):
```
Comment (moonso) [inline scout/export/variant.py]: You could skip this if by doing:

File: scout/export/variant.py [hunk 2]
```diff
@@ -135,13 +135,14 @@ def export_mt_variants(variants, sample_id):
...
+            for gene in variant['genes']:
```
Comment (moonso) [inline scout/export/variant.py]:
```suggestion
            for gene in variant.get('genes',[]):
```

**Comment by northwestwitch** [comment]
> Have you installed on stage?

No, I'll do it now

**Comment by northwestwitch** [comment]
> > Have you installed on stage?
> 
> No, I'll do it now

Done. Could you check now?

**Pull request merged**